common/exprParser/expression.js

```diff
@@ -6,7 +6,7 @@ const { toString: toStringUtils, isValid } = require('./helpers/utils')
 const { types } = require('./helpers/types')
 
 const operators = require('./helpers/operators')
-const { toParamsObj, getWherePerparedStatement } = require('./helpers/sqlUtils')
+const { toSqlPreparedStamentParams, getWherePerparedStatement } = require('./helpers/sqlUtils')
```

Comment:
there's a type toSqlPrepared**Stament**Params --> toSqlPrepared**Statement**Params

Reply:
ok let's postpone it